hawc/main/urls.py

```diff
@@ -43,6 +70,16 @@
     ),
     url(rf"^admin/{settings.ADMIN_URL_PREFIX}/", admin.site.urls),
     url(r"^selectable/", include("selectable.urls")),
+    url(
+        r"^openapi/$",
+        get_schema_view(
+            title="HAWC OpenAPI",
+            version="0.0.1",
```

Comment:
can we add a `__version__` to `hawc/__init__.py` and use that value here. I think it should be 0.1 since we've used that elsewhere, please confirm

Reply:
I tried it with a string parameter, [but it turns out that can only be done with modules.](https://docs.djangoproject.com/en/3.0/ref/urls/#include)

Reply:
Yes, it is 0.1 in the setup.cfg. I made the requested changes.